lib/modules/platform/utils/read-only-issue-body.ts

```diff
@@ -2,10 +2,10 @@ import { regEx } from '../../../util/regex';
 
 export function readOnlyIssueBody(body: string): string {
   return body
-    .replace(' only once you click their checkbox below', '')
-    .replace(' unless you click a checkbox below', '')
-    .replace(' To discard all commits and start over, click on a checkbox.', '')
-    .replace(regEx(/ Click (?:on |)a checkbox.*\./g), '')
+    .replace(' only once you select their checkbox below', '')
+    .replace(' unless you select a checkbox below', '')
+    .replace(' To discard all commits and start over, select a checkbox.', '')
+    .replace(regEx(/ Select (?:on |)a checkbox.*\./g), '')
```

Comment:
I guess we need to change this line of code, as we should never say:

> Select **on** a checkbox.

Reply:
"Select a checkbox"?

Reply:
Want me to change it to "Select **this** checkbox" then?